### ISSUE TITLE
isFunction helper

### ISSUE THREAD

**Issue by DimaGashko** [sha256/Pristine#23]
Why do you use [this helper](https://github.com/sha256/Pristine/blob/master/src/utils.js#L24)?

```js
export function isFunction(obj) {
    return !!(obj && obj.constructor && obj.call && obj.apply);
}
```

Why not just `typeof func === 'function'`?

**Comment by sha256** [comment]
typeof Object, typeof Date, and typeof String, etc all return 'function'

**Comment by DimaGashko** [comment]
Yes, them does... but that's because all of them are functions. And that's why ever the isFunction helper also says that them are functions

```js
function isFunction(obj) {
    return !!(obj && obj.constructor && obj.call && obj.apply);
}
undefined
isFunction(Object)
true
isFunction(String)
true
isFunction(Date)
true
```

![image](https://user-images.githubusercontent.com/20650419/91276284-fd094380-e789-11ea-8d6f-d7a059a090b8.png)

**Comment by sha256** [comment]
Hi @psmyrdek, could you please have a look since this function was added by you? :)
